python/lsst/ap/verify/testPipeline.py

```diff
@@ -0,0 +1,462 @@
+#
```

Comment:
Oh dear. You're actually the first person in 4.5 years to notice! 🤦

The `ap_verify` package is supposed to follow the usual [Science Pipelines conventions](https://developer.lsst.io/python/style.html#naming-conventions-for-science-pipelines), since it basically sits at the top of all the other pipeline code. The `ap_verify` _module_ can be excused in that it's supposed to be the interface (including the "main" function) of the `ap_verify` package, but `pipeline_driver` is definitely wrong.

Reply:
Written up as [DM-33746](https://jira.lsstcorp.org/browse/DM-33746).